src/libraries/System.Text.Json/src/System/Text/Json/JsonEncodedText.cs

```diff
@@ -73,16 +73,19 @@ private static JsonEncodedText TranscodeAndEncode(ReadOnlySpan<char> value, Java
             JsonWriterHelper.ValidateValue(value);
 
             int expectedByteCount = JsonReaderHelper.GetUtf8ByteCount(value);
+            Console.WriteLine("expectedByteCount = " + expectedByteCount.ToString());
```

Comment:
I believe this is something for debugging and you will revert it before merging.

Reply:
Yup. All changes to all the other files were debug changes. I reverted all of them